### PR TITLE
Add resolv job

### LINKED ISSUE #26: enhancement to add `resolv.conf` options set
Either modify search_domain job or create a new job to modify `resolv.conf` options set such as timeout, rotate, etc. 
Ref: https://www.freebsd.org/cgi/man.cgi?query=resolv.conf&manpath=SuSE+Linux/i386+11.3

### PR DESCRIPTION
**Pull request by ssurenr**
This job adds resolv.conf options set. Fixes #26

### CODE REVIEW

File: jobs/resolv/templates/pre-start.sh.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+sed -i '/^options/d' /etc/resolvconf/resolv.conf.d/tail
+
+<% p("resolv.options").each do | option | %>
+echo "options <%= "#{option}" %>" >> /etc/resolvconf/resolv.conf.d/tail
```
Comment (cppforlife) [inline jobs/resolv/templates/pre-start.sh.erb]: `"#{option}"` -> `<%= option %>`

File: jobs/resolv/spec [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: resolv
+
+templates:
+  pre-start.sh.erb: bin/pre-start
+
+properties:
+  resolv.options:
+    description: Set resolv.conf options set
```
Comment (cppforlife) [inline jobs/resolv/spec]: let's add `default: []`

**Comment by ssurenr** [comment]
Modified chages as suggested and tested. Good to go.

**Comment by ssurenr** [comment]
@cppforlife If you need anything else, let me know.

**Comment by cppforlife** [comment]
@ssurenr i've updated resolv job a bit, check it out.

**Comment by ssurenr** [comment]
Saw that. That's great. Planning to remove search_domain job?

2018-04-13 2:40 GMT+05:30 Dmitriy Kalinin <notifications@github.com>:

**Pull request merged**